packages/frontend-2/components/projects/Dashboard.vue

```diff
@@ -18,6 +18,9 @@
     </div>
 
     <PromoBannersWrapper v-if="promoBanners.length" :banners="promoBanners" />
+
+    <ProjectsDashboardDiscoverableWorkspaces v-if="isWorkspacesEnabled" />
```

Comment:
This page is already there on /dashboard, you can find it here: `packages/frontend-2/pages/dashboard.vue`